src/unordered.rs

```diff
@@ -1057,7 +1057,7 @@ fn bit_rev_twice_inv(nbits: u32, base_nbits: u32, i: usize) -> usize {
     bit_rev(nbits, i_rev)
 }
 
-#[cfg(test)]
+#[cfg(all(test, not(target_arch = "wasm32")))]
```

Comment:
could you remind us what the issue is for tests on wasm ?

Reply:
One of the libraries used for generating random values for the tests breaks on my linux machine.  I had found the exact dependency that caused it. It was such a long time ago that I forgot which was it. Those tests will run when I test for wasm. Meaning it will start a library that will break on my machine every time. My guess is that it thinks that my Linux machine is a mac.

This is why I had to disable those tests. I apologize for the issue caused by my hardware.

Since those tests couldn't be run in this machine, I can't guarantee that they will run well in your systems compiled to wasm. I will leave that to you.

Making these changes allowed a webpage to generate a server key and do computations on encrypted data. It may be a bit slower, but it didn't seem impactful from the user's perspective.